controllers/core/openstackversion_controller.go

```diff
@@ -240,10 +238,22 @@ func (r *OpenStackVersionReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			corev1beta1.OpenStackVersionMinorUpdateOVNControlplane,
 			corev1beta1.OpenStackVersionMinorUpdateReadyMessage)
 
+		// minor update for Dataplane OVN
+		if !openstack.DataplaneNodesetsOVNControllerDeployedCheck(instance, dataplaneNodesets) {
```

Comment:
here I would simply drop the Check from the end of the function name, as it will automatically suggest that it is true when OVN is deployed.